packages/angular/projects/ui-angular/src/lib/services/custom-components.service.ts

```diff
@@ -16,12 +17,4 @@ export class AuthPropService {
   public set customComponents(customComponents: CustomComponents) {
     this._customComponents = { ...this._customComponents, ...customComponents };
   }
-
-  public get props(): PropContext {
-    return this._props;
-  }
-
-  public set props(props: PropContext) {
-    this._props = { ...this._props, ...props };
-  }
```

Comment:
This was previously used to share props between subcomponents, but isn't needed currently as all the props are being passed to `amplify-authenticator` anyway. Will add back in as needed.

Reply:
Nice refactor